### DISCUSSION TITLE
Including Virtual-LTSP

### DISCUSSION THREAD

**Discussion by dashohoxha** [ltsp/ltsp#179] [Q&A]
What do you think about including these projects into https://github.com/ltsp:
- https://gitlab.com/Virtual-LTSP/VirtualBox
- https://gitlab.com/Virtual-LTSP/Packer

I think they are closely related and having them in the same place might create synergy.
I even had plans for running a LTSP server from a Docker container, but couldn't find time to work on that. Maybe I can try it again in the future.

**Comment by alkisg** [comment]
Hi Dashamir,

the first link seems to be specific to LTSP5; it'll need to be updated for the new LTSP. It'll also need a review, as it's trying to do some things that LTSP already does, but in a different way than what upstream recommends. And it appears to be customizing things for some specific use case? Will that be useful for all LTSP users?

Packer seems very interesting; maybe you could start by writing a how-to in the [community wiki](https://github.com/ltsp/community/wiki), and if LTSP users like it and start using it, then we could surely see how to include it upstream.

**Comment by dashohoxha** [comment]
> the first link seems to be specific to LTSP5

Of course, because it has not been updated yet to the new one.

> it's trying to do some things that LTSP already does, but in a different way than what upstream recommends

It was trying to solve some problems that LTSP5 could not solve properly.

> And it appears to be customizing things for some specific use case? Will that be useful for all LTSP users?

The primary goal is to run a LTSP server from your laptop. It has been useful to me for some time, maybe it could be useful for other people too.

> Packer seems very interesting; maybe you could start by writing a how-to in the community wiki, and if LTSP users like it and start using it, then we could surely see how to include it upstream.

It is not a matter of including it upstream, but rather just hosting it on the same project-space as LTSP, as a related project. Actually it is needed but the other project (Virtual-LTSP/VirtualBox) for generating a LinuxMint based Vagrant box (which also has edu packages pre-installed).
